language/grounding/intro-grounding.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<p><code> results of a data store in Vertex AI. </code> → <code> results of a data store in Vertex AI Search.</code></p><p>  </p><p><code>You'll ask a question about different services available in Vertex AI.</code> → <code>You'll ask a question about different services available in Google Cloud.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
Fixed in b15bbec. Thanks for catching these!